Formula/git-town.rb

```diff
@@ -16,9 +16,11 @@ class GitTown < Formula
 
   def install
     ENV["GOPATH"] = buildpath
-    (buildpath/"src/github.com/Originate").mkpath
-    ln_sf buildpath, buildpath/"src/github.com/Originate/git-town"
-    system "go", "build", "-o", bin/"git-town"
+    (buildpath/"src/github.com/git-town").mkpath
+    ln_sf buildpath, buildpath/"src/github.com/git-town/git-town"
```

Comment:
you can also deprecate the GOPATH usage too. (see #47627)

Reply:
Thanks so much for helping with this!! You are right. Removed this line since Git Town is now a Go module.